x/wasm/keeper/keeper.go

```diff
@@ -960,6 +960,26 @@ func (k Keeper) setAccessConfig(ctx sdk.Context, codeID uint64, caller sdk.AccAd
 	return nil
 }
 
+func (k *Keeper) HandleContractResponse(
```

Comment:
I don't think we should expose this, or at least expose it for tests only.

Reply:
tysm, I'll fix it

Reply:
Hi @jhernandezb , I have fixed this problem. Tysm for your reviews. I've learned a lot <3